kernel/kernel-auth-adapter/src/main/java/io/mosip/kernel/auth/defaultadapter/handler/AuthHandler.java

```diff
@@ -361,9 +399,9 @@ private void sendErrors(RoutingContext routingContext, List<ServiceError> errors
 			LOGGER.error(exception.getMessage());
 		}
 	}
-	
+
 	public String getContextUser() {
-		Authentication authentication=SecurityContextHolder.getContext().getAuthentication();
-		return authentication==null?DEFAULTADMIN_MOSIP_IO:authentication.getName();
+		Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+		return authentication == null ? DEFAULTADMIN_MOSIP_IO : authentication.getName();

```

Comment:
why should it give this user? why not throw an exception? or send empty? What's the use of this?